pypureclient/flasharray/FA_2_0/models/api_version_response.py

```diff
@@ -63,6 +63,24 @@ def __getattribute__(self, item):
         else:
             return value
 
+    def __getitem__(self, key):
+        if key not in self.attribute_map:
+            raise KeyError("Invalid key `{}` for `ApiVersionResponse`".format(key))
+        return object.__getattribute__(self, key)
+
+    def __setitem__(self, key, value):
+        if key not in self.attribute_map:
+            raise KeyError("Invalid key `{}` for `ApiVersionResponse`".format(key))
+        object.__setattr__(self, key, value)
+
+    def __delitem__(self, key):
+        if key not in self.attribute_map:
+            raise KeyError("Invalid key `{}` for `ApiVersionResponse`".format(key))
+        object.__delattr__(self, key)
+
+    def keys(self):
+        return self.attribute_map.keys()
+
```

Comment:
Is this related or unrelated to https://github.dev.purestorage.com/Pure1/python-sdk/pull/231

Reply:
Yes, those methods were introduced for every model. They allow to access properties in the dictionary-like way.